### DISCUSSION TITLE
Getting Started

### DISCUSSION THREAD

**Discussion by lrljoe** [joshhanley/livewire-dusk-testbench#3] [General]
Using this with Livewire v3, and it's great.

Couple of pointers that I ran into when first using this are:
1. Ensure that you have Chrome Driver present (and Chrome itself installed)
2. Create your "views" directory, **including** a layout file!
3. Ensure that you include your package service provider, and any dependency Service Providers in your TestCase (you don't need to include Laravel/Livewire here tho!)
4. Unless your package is returning full page components, make sure to remember to use a wrapping/parent Component!

# Common Gotchas
1. If using parallel testing, Dusk often misbehaves, avoid this by separating your test-suites, and not running Dusk tests in parallel.

**Comment by joshhanley** [comment]
Thanks! I think most of that should be in the README, but if it's not, feel free to add it 🙂
